corehq/apps/hqwebapp/static/hqwebapp/js/atwho.js

```diff
@@ -16,6 +18,7 @@ hqDefine('hqwebapp/js/atwho', function () {
      *  afterInsert: will be called after new value is selected.
      */
     var init = function($input, options) {
+        var replaceValue = options.replaceValue === undefined || options.replaceValue;
```

Comment:
This is meant to default `replaceValue` to true, right? How about requiring `replaceValue` to be explicitly supplied instead, to be more readable? There's a new-ish [assert properties util](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/assert_properties.js) you could use to do something like `assert(options, ['replaceValue'], [...])`.